src/plugins/components/PluginsList/PluginsList.tsx

```diff
@@ -132,15 +132,15 @@ const PluginList: React.FC<PluginListProps> = props => {
                 {maybe<React.ReactNode>(() => plugin.name, <Skeleton />)}
               </TableCell>
               <TableCell className={classes.colActive}>
-                {maybe<React.ReactNode>(
+                {/* {maybe<React.ReactNode>(
                   () => (
                     <StatusLabel
                       label={translateBoolean(plugin.active, intl)}
                       status={plugin.active ? "success" : "error"}
                     />
                   ),
                   <Skeleton />
-                )}
+                )} */}
```

Comment:
🚓

Reply:
This part of code overlaps with [plugin details pr](https://github.com/mirumee/saleor-dashboard/pull/1083) and types would fail miserably if it wasn't commented out. Will be removed in feature branch